scripts/f5-config.py

```diff
@@ -14,55 +14,155 @@
 # limitations under the License.
 #
 # (c) 2014, Kevin Carter <kevin.carter@rackspace.com>
-
 import argparse
 import json
 import os
 
 import netaddr
 
+
+PART = 'RPC'
+PREFIX_NAME = 'RPC'
+
 SNAT_POOL = (
-    'create ltm snatpool %(prefix_name)s_SNATPOOL { members replace-all-with {'
-    ' %(snat_pool_addresses)s%(route_domain)s } }'
+    '### CREATE SNATPOOL ###\n'
+    'create ltm snatpool /' + PART + '/' + PREFIX_NAME + '_SNATPOOL { members replace-all-with {'
+    ' %(snat_pool_addresses)s } }'
 )
 
-MONITORS = (
-    'create ltm monitor mysql %(prefix_name)s_MON_GALERA { count 0 database'
-    ' information_schema debug yes defaults-from mysql destination *:*'
-    ' interval 30 time-until-up 0 timeout 91 username haproxy }'
-)
+#Persistance Profile:
+PERSISTANCE = [
+    r'create ltm persistence source-addr /' + PART + '/' + PREFIX_NAME + '_PROF_PERSIST_IP {'
```

Comment:
Why are you using raw strings here? Those are most useful when trying to use `\`s as just `\` instead of having them used to escape something.

Especially below, you have `'\n'` as something other than a raw string and everything else just has `/`. Am I missing something?